### PR TITLE
Merge Weasyl API Documentation

### PR DESCRIPTION
**Pull request by kfkitsune**
Resolves issue #75.

Merges the weasyl/weasyl-apidocs repository contents into weasyl/weasyl, preserving the commit history.

### CODE REVIEW

File: weasyl-apidocs/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/build
+/source/*.pyc
```
Comment (charmander) [inline weasyl-apidocs/.gitignore]: Root `.gitignore` already includes `*.py[co]`.

**Comment by weykent** [comment]
Thanks for this! I was hoping to use [`git subtree`](https://github.com/git/git/blob/master/contrib/subtree/git-subtree.txt) to do the merge, though, so the history is preserved.

File: weasyl-apidocs/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+copyright = u'2014, weykent'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# version = release = vcversioner.find_version(
+#     root=os.path.dirname(here),
+#     version_file=None,
+# ).version
```
Comment (weykent) [inline weasyl-apidocs/source/conf.py]: Does this actually build with these missing?

File: weasyl-apidocs/source/conf.py [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+
+# The encoding of source files.
+#source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'Weasyl API'
+copyright = u'2014, weykent'
```
Comment (weykent) [inline weasyl-apidocs/source/conf.py]: Should probably bump this if we're making changes anyway.

File: weasyl-apidocs/source/conf.py [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+
+# If false, no module index is generated.
+#texinfo_domain_indices = True
+
+# How to display URL addresses: 'footnote', 'no', or 'inline'.
+#texinfo_show_urls = 'footnote'
+
+
+highlight_language = 'json'
+edit_on_github_project = 'Weasyl/weasyl-apidocs'
```
Comment (weykent) [inline weasyl-apidocs/source/conf.py]: This needs to be updated.

File: weasyl-apidocs/source/edit_on_github.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+__licence__ = 'BSD (3 clause)'
+
+
+def get_github_url(app, view, path):
+    return 'https://github.com/{project}/{view}/{branch}/source/{path}'.format(
+        project=app.config.edit_on_github_project,
+        view=view,
+        branch=app.config.edit_on_github_branch,
+        path=path)
```
Comment (weykent) [inline weasyl-apidocs/source/edit_on_github.py]: This will need an extra `weasyl-apidocs/` in there since it's not in the repo root anymore.

**Pull request merged**